### ISSUE TITLE
Tab inset regression in 0.13.6

### ISSUE THREAD

**Issue by wahern** [yue/yue#162]
Yue Version: 0.13.6
Operating System: macOS

The Tab inset fix in 0.13.6 seems to have had the opposite effect as intended. I have a Tab widget which takes up the entire width and height of a window. Everything fit properly up to 0.13.5. With 0.13.6 the Tab itself is askew in the window, with no inset on the left and bottom, and too much inset on the right. Worse, views inside the Tab overflow on the right and bottom, even causing clipping.

This happens whether I assign the Tab directly as the window content, or if I nest the Tab inside a container and assign the container as the window content. (I tried the latter to see if I could compensate with some padding or margin in a parent container, but that didn't quite work because the tab itself reflowed differently on size change than in the original layout, so I couldn't come up with consistent compensation scheme.)

**Comment by zcbenz** [comment]
I confirm the bug.

Code snippet used for testing:

```lua
local gui = require('yue.gui')

local win = gui.Window.create{}
win.onclose = function() gui.MessageLoop.quit() end

local tab = gui.Tab.create()
win:setcontentview(tab)

local content = gui.Container.create()
content:setbackgroundcolor('#693')
tab:addpage('Tab', content)

win:setcontentsize{width=400, height=400}
win:center()
win:activate()

gui.MessageLoop.run()
```

**Comment by zcbenz** [comment]
Closed with 4bb3eaa73a2142eebc466d3e64952c0efb482391.
